content/Arduino Cloud/General Cloud/If-your-Arduino-device-can-t-connect-to-a-Wi-Fi-network.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "If your Arduino device can’t connect to a Wi-Fi network"
+---
+
+Learn how to troubleshoot Wi-Fi connectivity issues if your Arduino device can't connect to a network. Check the sections below to identify and resolve potential issues.
+
+## 1. Connect to a 2.4 GHz network
+
+Learn more about it [here](https://support.arduino.cc/hc/en-us/articles/14144656051740-Supported-Wi-Fi-bands)
+
+## 2. Avoid common network names
+
+Make sure your Wi-Fi network has a distinct name. Avoid using generic names like 'default' or 'home' to prevent your device from accidentally connecting to a different network with the same name, which can lead to performance issues.
+
+## 3. Reset network credentials for your Thing
+
+### Verifying or changing network credentials in Arduino Cloud
+
+If the Wi-Fi connection is not working, you might need to reset the [network credentials](https://support.arduino.cc/hc/en-us/articles/360016495559-Add-and-connect-a-device-to-Arduino-Cloud#:~:text=Click%20the%20Network,adding%20the%20device) for your Thing in Arduino Cloud.
+
+1. Navigate to the [Thing](https://app.arduino.cc/things) that is experiencing Wi-Fi issues.
+
+1. Under **Network**, click the button labeled "Change"
+
+1. Re-enter your Wi-Fi credentials with your Wi-Fi Name (SSID) and Password (Encryption Key). Ensure there are no typos and the credentials match those set on your router.
+
+1. Open the sketch and click the **Upload** button to properly apply the new network settings to your device.
+
+### Veryfing or changing the credentials defined in the sketch
+
+To verify or change the network credentials directly defined in a sketch, whether the sketch was created in Arduino Cloud or IDE, follow these steps:
+
+1. Locate the section in your Arduino sketch where the network credentials are defined. This is typically found in the main sketch file or inside an `arduino_secrets.h` file.
```

Comment:
For Arduino Cloud Editor users, they will be in the "**Secret**" tab instead of `arduino_secrets.h`:

https://docs.arduino.cc/arduino-cloud/cloud-editor/store-your-sensitive-data-safely-when-sharing